pkg/jobs/update.go

```diff
@@ -307,6 +308,65 @@ WHERE id = $1
 			}
 		}
 	}
+	if v.AtLeast(clusterversion.V25_1_AddJobsColumns.Version()) {
+
+		vals := []interface{}{j.ID()}
+
+		var update strings.Builder
+
+		if payloadBytes != nil {
+			if beforePayload.Description != payload.Description {
+				if update.Len() > 0 {
+					update.WriteString(", ")
+				}
+				vals = append(vals, payload.Description)
+				fmt.Fprintf(&update, "description = $%d", len(vals))
+			}
+
+			if joined := strings.Join(payload.Statement, "; "); strings.Join(beforePayload.Statement, "; ") != joined {
+				if update.Len() > 0 {
+					update.WriteString(", ")
+				}
+				vals = append(vals, joined)
+				fmt.Fprintf(&update, "statement = $%d", len(vals))
+			}
+
+			if beforePayload.UsernameProto.Decode() != payload.UsernameProto.Decode() {
+				if update.Len() > 0 {
+					update.WriteString(", ")
+				}
+				vals = append(vals, payload.UsernameProto.Decode().Normalized())
+				fmt.Fprintf(&update, "owner = $%d", len(vals))
+			}
+
+			if beforePayload.Error != payload.Error {
```

Comment:
`FinalResumeError` is machine-readable rather than human readable so it can stay in a proto in `job_info` i.e. where it is now; it isn't and won't be used by `SHOW`